python/tchannel/tornado/tchannel.py

```diff
@@ -64,7 +66,7 @@ class TChannel(object):
     }
 
     def __init__(self, name, hostport=None, process_name=None,
-                 known_peers=None):
+                 known_peers=None, zipkin=False):
```

Comment:
Zipkin is one instance of a tracer we can turn on, but it's not the only one. I'd refrain from explicitly mentioning Zipkin here and use something like `trace=True` instead.